covariance/covarianceXsec.cpp

```diff
@@ -829,7 +829,7 @@ void covarianceXsec::initParams(double fScale) {
 
 // ********************************************
 // Get the likelihood for moving the cross-section parameters to these values
-double covarianceXsec::GetLikelihood() {
+double covarianceXsec::getLikelihood() {
```

Comment:
I only changed these because in covarianceBase.cpp and everywhere else except here it's getLikelihood, not GetLikelihood. So make everything GetLikelihood?